### ISSUE TITLE
Swift Unit Tests stopped working.

### ISSUE THREAD

**Issue by OskarPeredo** [GoogleCloudPlatform/ios-docs-samples#70]
```
## OS
ProductName:	Mac OS X
ProductVersion:	10.13.1
BuildVersion:	17B1003
### POD
1.4.0.beta.1
### Project
App/Podfile
/Users/Oscar/documents/projects/test/App
```

---

### Problem | Feature

Unit Tests stop working after gRPC has been installed.

Although the speech recognition seems to work fine so far, i've lost the ability to run Unit Tests.

The project builds correctly, however when I click on the run tests little arrow the following error is displayed:

    Swift Compiler Error Group

    /Users/Oscar/Documents/Projects/test/App/google/api/Annotations.pbobjc.h:13:10: 'GPBProtocolBuffers.h' file not found

    Could not build Objective-C module 'googleapis'

IMPORTANT: I'm not using gRPC anywhere in my testing code (I could run tests normally before installing gRPC, and I've not added any new tests)

**Comment by timburks** [comment]
Hi Oscar,

How are you running these unit tests? 

The gRPC support in these samples requires Objective-C. Are any other Objective-C dependencies being built for your tests?

**Comment by OskarPeredo** [comment]
Hey, sorry for the late reply. I had to focus on a different branch of my project.

Turns out that even though the newest version of the repository automatically adds the preprocessor macro to the target file. It doesn't do it for the test target. (So I added this flag myself)

`GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1
`

I also ended changing the inheritance for my test target on the pod file.

```
target 'MyTests' do

  inherit! :complete

end

```
Thanks for your help.
